docs/installation.rst

```diff
@@ -203,10 +205,10 @@ We can install ``msprime`` easily using pip::
 
     $ python3 -m pip install msprime
 
-(It is generally better to use ``python3 -m pip`` rather than call ``pip``
+(It is generally better to use ``python3 -m pip`` or call ``pip3`` rather than
```

Comment:
I'd delete the "or call pip3" bit here - the distinction is between using ``python -m pip`` rather than the binary ``pip`` from your PATH because at least then you know you're installing to the right version of Python. Mac users in particular often have a large number of independent installations of Python of varying levels of brokenness. This saves a bit of confusion for them.